index.html

```diff
@@ -1,199 +1,315 @@
+            </h1>
+            <div class="d-first-paragraph">
+                <p class="first-paragraph"> I can help you build a product, feature or website Look through some of my work
+                and experience! If you like what you see and have a project you need coded,
+                don’t hesitate to contact me.</p>
+            </div>
+            
+        </div>
+            <ul class="social-media">
+                <li><a href="#"><img src="images/twitter.svg" alt="twitter"></a></li>
+                <li><a href="#"><img src="images/linkedin.svg" alt="linkeldin"></a></li>
+                <li><a href="#"><img src="images/medium.svg" alt="medium"></a></li>
+                <li><a href="#"><img src="images/github.svg" alt="github"></a></li>
+                <li><a href="#"><img src="images/angellist.svg" alt="angellist"></a></li>
+            </ul>
+            <div class="down-button">
+                <img src="images/down-button.svg" alt="down-button">
+            </div>
         </main>
+
+        </section>
+ 
+    <div class="my-recent-works">
+        <h1 class="domain">My Recent <br class="break"> Works</h1>
+    </div>
+
     
-            <h1 class="Domain">My Recent <br> Works</h1>
-            <div class="little-line"></div>
-        
-            
+    <div class="little-line"></div>
+    
+     <div class="all-card-holders">
         <article class="card-holder">
             <section>
                 <div class="card">
-                <div class="empty-box">
-
-                </div>
-                <div class="multi-post-stories">
-                    <h3>Multi-Post Stories Gain + Glory</h3>
-                    <div class="programing-languages">
-                    <ul>
-                        <li>Ruby on Rails</li>
-                        <li>CSS</li>
-                        <li>Javascript</li>
-                        <li>HTML</li>
-                </ul>
-                </div>
-                <button class="green-button">See project</button>
-                </div>
-                
+                    <div class="empty-box">
+    
+                    </div>
+                    <div class="multi-post-stories">
+                        <h3>Multi-Post Stories Gain + Glory</h3>
+                        <div class="programing-languages">
+                            <ul>
+                                <li>Ruby on Rails</li>
+                                <li>CSS</li>
+                                <li>Javascript</li>
+                                <li>HTML</li>
+                            </ul>
+                        </div>
+                        <button class="green-button">See project</button>
+                    </div>
+    
             </section>
-            <article class="card-holder">
-                <section>
-                    <div class="card">
+        </article>
+        <article class="card-holder">
+            <section>
```

Comment:
- [ ] According to your previouse reviewer you have used the <article> tag for the cards, which is a good practice for semantic HTML. However, it is unnecessary to have a <section> tag inside each <article> tag. Please consider removing the <section> tags within the <article> tags to simplify the code structure.